pkg/client/search.go

```diff
@@ -38,7 +33,7 @@ func (c *Client) Search(queryName string, queryFor string, params map[string]int
 }
 
 func (c *Client) SearchEvents(params map[string]interface{}, fields []string) ([]storage.Event, error) {
-	response, err := c.Search("FindEvents", "events", params, fields)
+	response, err := c.Search("events", params, fields)
```

Comment:
Could the consts from `pkg/client/graphql.go` be exported & used here? Usage wise, same applies for `SearchEventReceivers()` & `SearchEventReceiverGroups()` in this same file.